include/dlr.h

```diff
@@ -48,7 +48,7 @@ typedef void* (*DLRMemalignFunctionPtr)(size_t, size_t);
 #endif
 
 /*!
- \brief Creates a DLR model.
+ \brief Creates a DLR model by searching in given directory path.
```

Comment:
It is not only a directory might be given, but colon separated list of files.

Reply:
updated